public/instances.json

```diff
@@ -0,0 +1,72 @@
+{
+  "instances": [
+    {
+      "name": "Contrib",
+      "image": "/images/instances/contrib.png",
+      "languages": ["en", "fr"],
+      "countries": ["EU", "US"],
+      "maintainers": [],
+      "stats": {
+        "services": "285",
+        "documents": "612"
+      },
+      "subscribeURL": "https://opentermsarchive.org/subscribe"
+    },
+    {
+      "name": "France",
+      "image": "/images/instances/france.png",
+      "languages": ["fr"],
+      "countries": ["FR"],
```

Comment:
Please use consistent casing between the two codes 🙂

Reply:
Thanks for objecting @martinratinaud! 👍 Thanks to you I learned today that:

- “ISO 639-2 recommends use of the language codes in lower case (but they should be considered case-insensitive and are unique codes regardless of case)”, according to the [US Library of Congress](https://www.loc.gov/standards/iso639-2/faq.html#21). The standard itself is not available for free, so I cannot double-check.
- ISO 3166-1:2020 specifies that “code element shall consist only of […] LATIN CAPITAL LETTER A through LATIN CAPITAL LETTER Z […]”, according to the standard itself (or rather the [leaked version](https://cdn.standards.iteh.ai/samples/72482/cbb6318e772a4f2284bdb0bd51f29017/ISO-3166-1-2020.pdf) since, once more, the standard is not freely accessible).

See also https://github.com/stefangabos/world_countries/issues/50.

I definitely consider jurisdictions as “countries”.